### PR TITLE
Chapter 2 finite volume draft

### PR DESCRIPTION
**Pull request by LeidongX**

### CODE REVIEW

File: tex/chapters/chapter3.tex [hunk 1]
```diff
@@ -1,5 +1,69 @@
...
-
-\chapter{Finite Volume Method of 1D Time Dependent Heat Equation}
+
+\cleardoublepage
+
+
+\chapter{Finite Volume Method of 1D Time Dependent Heat Equation}
+
+\section{Governing Equation}
+An one-dimensional nonlinear time dependent heat equation is defined as:
```
Comment (robertsj) [inline tex/chapters/chapter3.tex]: The one-dimensional, nonlinear, time-dependent...as (no colon)

File: tex/chapters/chapter3.tex [hunk 2]
```diff
@@ -1,5 +1,69 @@
...
+\begin{equation}
+\frac{\partial u}{\partial t} = \alpha(x,u) \frac{\partial^2 u}{\partial x^2} + S(x,t)
```
Comment (robertsj) [inline tex/chapters/chapter3.tex]: add `\, ,` after `S(x, t)` to a nice looking comma

File: tex/chapters/chapter3.tex [hunk 3]
```diff
@@ -1,5 +1,69 @@
...
+\end{equation}
+where thermal diffusivity $\alpha$ is linearly dependent on density ($\rho$), heat capacity(C), and thermal conductivity ($\kappa$) by:
```
Comment (robertsj) [inline tex/chapters/chapter3.tex]: where *the* thermal...

**Comment by robertsj** [comment]
That said, this will be about the right amount of text for me to review (as long as it won't require too much wordsmithing).

File: tex/chapters/chapter3.tex (hunk 1 above)
Comment (LeidongX) [reply tex/chapters/chapter3.tex]: As a NNS, it takes me a long time to write in delicate words and elegant style. I wrote this for check if the discretization is correct. I will put more effect on formatting and standardizing my writing when I formally start drafting my dissertation.  So far, I wonder if it is mathematically alright. Therefore, I can implement FVM at the solver. 



> The one-dimensional, nonlinear, time-dependent...as (no colon)

File: tex/chapters/chapter3.tex (hunk 2 above)
Comment (LeidongX) [reply tex/chapters/chapter3.tex]: Added in e8f22febc73c4caf312ef518aa2e9d043fa4d740

File: tex/chapters/chapter3.tex (hunk 3 above)
Comment (LeidongX) [reply tex/chapters/chapter3.tex]: correct in e8f22febc73c4caf312ef518aa2e9d043fa4d740

**Pull request closed without merging**